### PR TITLE
Continue prompt fix

### PR DESCRIPTION
**Pull request by sw4rm3r**

### CODE REVIEW

File: .env.template
Comment (nponeccop) [inline .env.template:1]: This change shouldn't be in this PR

File: .gitignore
Comment (nponeccop) [inline .gitignore:1]: This one isnt related to the prompt fix either

**Review by nponeccop — CHANGES_REQUESTED** [review]
Remove the irrelevant changes and submit them in a separate PR

**Comment by nponeccop** [comment]
@sw4rm3r Resolve the conflicts

**Comment by p-i-** [comment]
This is a mass message from the AutoGPT core team.
Our apologies for the ongoing delay in processing PRs.
This is because we are re-architecting the AutoGPT core!

For more details (and for infor on joining our Discord), please refer to:
  https://github.com/Significant-Gravitas/Auto-GPT/wiki/Architecting

**Comment by Pwuts** [comment]
* No description
* Invalid changes (deleting `.env.template`)
* No response to review
* Conflicts with base branch

Thanks for submitting, unfortunately we can't process this. Feel free to contribute again, but be sure to fill out the PR description (template).

**Pull request closed without merging**